internal/controller/persistentvolume/persistentvolume.go

```diff
@@ -126,19 +124,8 @@ func (r *ReconcilePersistentVolume) getCredentials(
 	return cr, nil
 }
 
-func checkStaticVolume(pv *corev1.PersistentVolume) (bool, error) {
-	static := false
-	var err error
-
-	staticVol := pv.Spec.CSI.VolumeAttributes["staticVolume"]
-	if staticVol != "" {
-		static, err = strconv.ParseBool(staticVol)
-		if err != nil {
-			return false, fmt.Errorf("failed to parse preProvisionedVolume: %w", err)
-		}
-	}
-
-	return static, nil
+func checkStaticVolume(pv *corev1.PersistentVolume) bool {
+	return pv.Spec.CSI.VolumeAttributes["staticVolume"] == "true"
```

Comment:
So we are defaulting to `false`, if the string value is not `true` (no need to check for invalid value)? 
This may cause problems, when do the same check during nodeStage.

How about we reuse function from https://github.com/ceph/ceph-csi/blob/f0b8a3f626ecb46a0fe0f22e0254383e5fbfd82f/internal/rbd/nodeserver.go#L116. ?

Reply:
@Rakshith-R on your queries:
1) anything other than `true` should be false.  
2) Regarding `nodeServer` we use seperate function now which is `IsStaticVolume` , so kept this only in controller. 

Does that answer your queries?

Reply:
okay, am fine with the change. 

@humblec  please satisfy the linter
```internal/controller/persistentvolume/persistentvolume.go:21: File is not `gofumpt`-ed (gofumpt)```

Reply:
@Rakshith-R done.. thanks .

Reply:
while I run `gofmt` locally I dont see any error, but fails here in CI..  :thinking:

Reply:
@humblec 
run ` gofmt -s -w persistentvolume.go` on the file ?

Reply:
@Rakshith-R thats what I was doing :) ```[hchiramm@localhost ceph-csi]$ gofmt -s -w internal/controller/persistentvolume/persistentvolume.go
[hchiramm@localhost ceph-csi]$ git diff
[hchiramm@localhost ceph-csi]$ 
```

Reply:
@Rakshith-R ooc, if you fetch the PR and run `gogmt` do you see any error?